prow/tide/tide.go

```diff
@@ -67,6 +66,12 @@ type contextChecker interface {
 	MissingRequiredContexts([]string) []string
 }
 
+type changedFilesProvider interface {
+	BatchChanges([]PullRequest) config.ChangedFilesProvider
```

Comment:
Nit: It is not necessary to make the members of a private interface that is only implemented in the same package public

Reply:
Thank you for the review. I have updated the PR.